protocol-definitions/MC.yaml

```diff
@@ -328,3 +328,58 @@ methods:
           since: 2.0
           doc: |
             Latest TimedMemberState of the member, serialized as JSON.
+  - id: 12
+    name: getClusterState
+    since: 2.0
+    doc: |
+      Gets the current state of a cluster.
+    request:
+      retryable: true
+      acquiresResource: false
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: currentState
+          type: byte
+          nullable: false
+          since: 2.0
+          doc: |
+            Current state of the cluster:
+            0 - ACTIVE
+            1 - NO_MIGRATION
+            2 - FROZEN
+            3 - PASSIVE
+            4 - IN_TRANSITION (not allowed)
+  - id: 13
+    name: shutdownCluster
+    since: 2.0
+    doc: |
+      Shuts down the cluster.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: -1
+    response: {}
+  - id: 14
+    name: changeClusterVersion
+    since: 2.0
+    doc: |
+      Changes the cluster version.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: -1
+      params:
+        - name: majorVersion
+          type: byte
+          nullable: false
+          since: 2.0
+          doc: |
+            Major component of the new cluster version.
+        - name: minorVersion
+          type: byte
```

Comment:
does it need to work with customer specific release umbers or beta releases, an e.g. 4.1-Visa ? Is byte enough?

Reply:
we are only interested in major and minor version, as this is the cluster version, not member version. so, yes, byte is enough.